x-pack/plugins/security_solution/server/lib/telemetry/helpers.test.ts

```diff
@@ -858,8 +859,12 @@ describe('test metrics response to value list meta data', () => {
         },
       },
     };
-    const response = metricsResponseToValueListMetaData(stubMetricResponses);
+    const response = formatValueListMetaData(stubMetricResponses, stubClusterInfo, stubLicenseInfo);
     expect(response).toEqual({
+      '@timestamp': '2023-01-30T00:00:00.000Z',
+      cluster_uuid: '5Pr5PXRQQpGJUTn0czAvKQ',
```

Comment:
Can you confirm these don't belong to a customer deployment, and where you got them from?

Reply:
Yup, got them the existing mock: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/telemetry/__mocks__/index.ts#L43. I don't see this cluster id being used anywhere either.